cmd/simplemon/main.go

```diff
@@ -0,0 +1,129 @@
+package main
+
+import (
+	"context"
```

Comment:
This is the start point of the application.

Into `cmd/simplemon/` folder has a lot of files, but to keep in simple I will try to generate all that have relation with API development into this folder!